README.md

```diff
@@ -1,38 +1,343 @@
 # <ins>Shana</ins>
 
-Shana is a bot with many functions, its initial idea was to completely replace Dyno, as such, to make it more reliable and
+Shana is a bot with many functions, its initial idea was to completely replace Dyno, as such, to make it more reliable
```

Comment:
This paragraph doesn't make sense to me, The first 3/4 of it does but I can't figure out your intent with the last sentence.